merlin/models/tf/core/transformations.py

```diff
@@ -40,6 +40,104 @@
 COUNT = utils.COUNT
 
 
+@tf.keras.utils.register_keras_serializable(package="merlin.models")
+class FeaturesTensorTypeConversion(TabularBlock):
+    """Base class to convert the tensor type of features provided in the schema
+
+    Parameters
+        ----------
+        schema : Optional[Schema], optional
+            The schema with the columns that will be transformed, by default None
+    """
+
+    def __init__(self, schema: Optional[Schema] = None, **kwargs):
+        super().__init__(**kwargs)
+        self.column_names = None
+        if schema is not None:
+            self.column_names = schema.column_names
+
+    def call(self, inputs: TabularData, **kwargs) -> TabularData:
```

Comment:
If this is intended to be a base class, is it safe to remove this call method?

Reply:
Sure, the base class `FeaturesTensorTypeConversion` should not have the `call()` method, as it need to be defined in child classes.